src/image-handler.ts

```diff
@@ -36,14 +36,16 @@ export async function moveImageToCrcCluster(image: ImageInfo): Promise<void> {
       name = name + ':' + image.tag;
     }
     await extensionApi.containerEngine.saveImage(image.engineId, name, filename);
-    await runCliCommand(getPodmanCli(), [
-      '--url',
-      `unix://${os.homedir()}/.crc/machines/crc/docker.sock`,
+    const result = await runCliCommand(getPodmanCli(), [
+      '--url=ssh://core@127.0.0.1:2222/run/podman/podman.sock',
```

Comment:
for now this can be hardcoded, but perhaps it is best to actually get this from the `podman-env` command (improvement task).